.github/workflows/ff-bleeding.yml

```diff
@@ -0,0 +1,39 @@
+name: Fast-forward bleeding
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+env:
+  FF_ISSUE_TITLE: "No-FF: bleeding is behind master"
+  GITHUB_TOKEN: ${{ github.token }}
+
+jobs:
+  release-tag-verification:
```

Comment:
Should probably be named fast-forward-bleeding

Reply:
Oops, indeed. Thank you.